warehouse/static/html/mirror.html

```diff
@@ -5,7 +5,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>Mirror of https://pypi.org</title>
 
-    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,400italic,600,600italic,700,700italic|Source+Code+Pro:500">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,400normal,600,600normal,700,700normal|Source+Code+Pro:500">
```

Comment:
No, please not yet, this is still **WIP**.

I'm away from my main workstation, I did not test the solution yet. I saw the issue, openend a PR (kinda saying: I'd like to take care of this as my first contribution).
I will test it as soon as I get back, clean it up, add needed screenshots, then it can be reviewd.